File: Questao11/OverflowUnderflow.java
public class OverflowUnderflow{
    public static void main(String []args){
        int overflow = 2147483647;
        System.out.println("Overflow: "+ (overflow + 1));
        int underflow = -2147483648;
        System.out.println("Underflow: "+ (underflow - 1));
    }
}

File: Questao11/OverflowUnderflow.c
#include <stdio.h>

int main()
{
    int overflow = 2147483647;
    printf("Overflow: %d\n", (overflow + 1));
    int underflow = -2147483648;
    printf("Underflow: %d\n", (underflow - 1));

    return 0;
}